bin/main.dart

```diff
@@ -11,7 +11,7 @@ import 'package:pana/pana.dart';
 final _gray = '\u001b[1;30m';
 final _none = '\u001b[0m';
 
-String gray(text) => "$_gray$text$_none";
```

Comment:
I'm using `Terminal.app` in `Pro` mode with semi-transparent background, in which `30m` is not visible, `37m` is better. I guess you are using it in either totally black or white background?